messages/org.json

```diff
@@ -1,6 +0,0 @@
-{
-  "commandDescription": "print a greeting and your org IDs",
```

Comment:
the messages were never being used, it might've been a leftover from when it was in a lerna project?